lib/action_controller/serialization.rb

```diff
@@ -30,7 +30,7 @@ def get_serializer(resource, options = {})
         serializable_resource.serialization_scope ||= serialization_scope
         serializable_resource.serialization_scope_name = _serialization_scope
         begin
-          serializable_resource.adapter
+          serializable_resource.tap(&:adapter)
```

Comment:
Could you explain the rationale here?
`serializable_resource.tap(&:adapter)` returns `serializable_resource` but calls the `serializable_resource.adapter` method in the meantime. Is this to ensure the serializers get built and the `CollectionSerializer::NoSerializerError` gets raised if necessary?